cmd/goclient/goclient.go

```diff
@@ -110,14 +111,14 @@ func main() {
 		argi++
 	}
 
-	var client *aptos.Client
+	var client *aptos_go_sdk.Client
```

Comment:
It feels silly to say "go SDK" here. That's from the perspective of Aptos publishing a thing, but not needed for a developer using it

Reply:
Woops.  I thought it was just aptos, let me fix that